From 2df96c458b98e9eb994970a8040972a1e22636dd Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 22 Oct 2024 15:22:18 +0100
Subject: [PATCH] build: Remove Jansson dependency

After previous changes, this library is no longer used.  We have
switched to json-c, for better compatibility with libvirt.

(cherry picked from commit e6dcf7e3a7e9170978e57ce6df1b34f92fac5ae3)
---
 m4/guestfs-libraries.m4 | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/m4/guestfs-libraries.m4 b/m4/guestfs-libraries.m4
index b76511982..80f9425f0 100644
--- a/m4/guestfs-libraries.m4
+++ b/m4/guestfs-libraries.m4
@@ -164,9 +164,6 @@ LIBS="$LIBS $LIBXML2_LIBS"
 AC_CHECK_FUNCS([xmlBufferDetach])
 LIBS="$old_LIBS"
 
-dnl Check for Jansson JSON library (required).
-PKG_CHECK_MODULES([JANSSON], [jansson >= 2.7])
-
 dnl Check for JSON-C library (required).
 PKG_CHECK_MODULES([JSON_C], [json-c >= 0.14])
 
